internal/codespaces/codespaces.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/cli/cli/v2/internal/codespaces/connection"
 )
 
+// codespaceStatePollingBackoff is the delay between state polls while waiting for codespaces to become
+// available. It's only exposed so that it can be shortened for testing, otherwise it should not be changed
+var codespaceStatePollingBackoff = backoff.NewConstantBackOff(10 * time.Second)
```

Comment:
Not sure if we want to follow what the extension does but if so, it starts off at 1 second with an exponential delay factor of 1.02.

Reply:
Oh nice callout, updated to match